src/ui/main.js

```diff
@@ -1,29 +1,47 @@
-import { React, useState, useCallback } from "react";
+import { StrictMode, React, useState, useCallback } from "react";
```

Comment:
This is a good catch. I should start using StrictMode all the time, but it's still a relatively new thing I think.

Reply:
@semmypurewal I imported it, but it seems that I didn't actually add it to the root... whoops.